File: Resueltos_por_competidor/Thomas_Molina/Bit.java
/*
 * Autor: Thomas Molina
 * Problema: A. Bit++
 * Juez online: Codeforces
 * Veredicto: Accepted
 * URL:https://codeforces.com/problemset/problem/282/A
 */


import java.util.Scanner;

public class Bit {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int contadorP = 0;
		int contadorN = 0;
		
		for (int i = 0 ; i<n; i++) {
			String a = sc.next();
			if (a.equals("X++")|a.equals("++X")) {
				contadorP++;
			}
			else if (a.equals("X--")|a.equals("--X")) {
				contadorN++;
				
			}
		}
		
		System.out.println(contadorP-contadorN);

	}

}

File: Resueltos_por_competidor/Carlos_Cobaleda/Bit.cpp
/*   
Autor: Carlos Eduardo Cobaleda Moreno
Problema: Bit++
Juez online: Codeforces
Veredicto: Accepted
URL: https://codeforces.com/contest/282/problem/A
*/

#include <bits/stdc++.h>
#include <string>
using namespace std;
 
int main() {
    int t; cin >> t;
    string sum = "++";
    string rest = "--";
    int x = 0;
    for (int i = 0; i < t; i++) {
        string line; cin >> line;
        if (line == "X++" || line == "++X") {
            x++;
        }
        if (line == "X--" || line == "--X") {
            x--;
        }
    }
 
    cout << x << endl;
 
    return 0;
}
